sample_workspace.py

```diff
@@ -0,0 +1,49 @@
+"""Sample workflow launch with an attached file."""
```

Comment:
Not sure about the goal of this sample and also I don't know how to run it.
It requires some extra info in the readme file.